backend/app/views/spree/admin/promotion_codes/index.csv.ruby

```diff
@@ -1,6 +1,6 @@
 CSV.generate do |csv|
   csv << ['Code']
-  @promotion.codes.order(:id).each do |code|
-    csv << [code.value]
+  @promotion.codes.order(:id).pluck(:value).find_each do |value|
```

Comment:
Unfortunately, `find_each` doesn't work here, because `pluck` returns an array. Would `.each` not work, now that we con't create AR objects but simple strings?